### PR TITLE
Navbar adjustments

### PR DESCRIPTION
**Pull request by rdvhoorn**
Added partners in navbar
Made dropdown hoverable in desktop and clickable in mobile

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -18,65 +18,95 @@
     <link rel="icon" type="image/png" href="/img/favicon.png"/>
     <title ng-bind="title + ' - H.S.A. Confluente'">H.S.A. Confluente</title>
 
+    <!-- Script to make the navbar hoverable in desktop format -->
```
Comment (AaronSaam) [inline src/index.html:21]: Not really a fan of having the javascript inside of the html file. But it works ;)

File: src/index.html [hunk 2]
```diff
@@ -18,65 +18,95 @@
...
+    <script>
+        $('.dropdown-toggle').click(function(e) {
+            if ($(document).width() > 768) {
+                e.preventDefault();
+
+                var url = $(this).attr('href');
+
+
+                if (url !== '#') {
```
Comment (Cookierock) [inline src/index.html]: Is this necessary? I mean, a '#' shouldn't break things right? 😅

File: src/index.html [hunk 3]
```diff
@@ -18,65 +18,95 @@
     <link rel="icon" type="image/png" href="/img/favicon.png"/>
     <title ng-bind="title + ' - H.S.A. Confluente'">H.S.A. Confluente</title>
 
+    <!-- Script to make the navbar hoverable in desktop format -->
+    <script>
+        $('.dropdown-toggle').click(function(e) {
```
Comment (AaronSaam) [inline src/index.html:23]: What does this script add exactly? All browsers don't see a '#' as a link to a new page in my experience 😬

File: src/css/style.css [hunk 4]
```diff
@@ -93,6 +93,14 @@ a {
 
 }
 
+@media only screen and (min-width: 768px) {
+
+    /* make dropdown menu's hoverable on desktop */
+    .dropdown:hover .dropdown-menu {
```
Comment (AaronSaam) [inline src/css/style.css:99]: This works, to also give the parent the 'active' background + let this work on a small window width, I personally think it is better to mimic the standard behaviour, i.e. adding the class 'open' to .dropdown.

**Review by AaronSaam — CHANGES_REQUESTED** [review]
It works, but I have a few questions.
Nice job nonetheless ;)

**Pull request closed without merging**